graph_nets/demos_tf2/shortest_path.ipynb

```diff
@@ -440,15 +474,15 @@
     "  \"\"\"\n",
     "  if not use_nodes and not use_edges:\n",
     "    raise ValueError(\"Nodes or edges (or both) must be used\")\n",
-    "  tdds = utils_np.graphs_tuple_to_data_dicts(target)\n",
-    "  odds = utils_np.graphs_tuple_to_data_dicts(output)\n",
+    "\n",
+    "  graphs_in_touple=target.globals.shape[0]\n",
```

Comment:
Use `target.n_node.shape[0]` instead

Reply:
But actually, from a quick check this seems to just work without any changes to this method.